test_contour_labels_strict_external flaky CI failure

```
FAILED tests/core/test_imagedata_filters.py::test_contour_labels_strict_external - assert 1.1244114842265844 < (1.4501817040145397 / 1.5)
```

Seen this fail on CI a few times in the merge queue

Saw it again:

```
FAILED tests/core/test_imagedata_filters.py::test_contour_labels_strict_external - assert 1.201873050071299 < (0.9461193853057921 / 1.5)
```

Linux py3.13, VTK 9.5.2 (latest numpy)

--- a/tests/core/test_imagedata_filters.py
+++ b/tests/core/test_imagedata_filters.py
@@ -2,7 +2,6 @@ from __future__ import annotations
 
 import operator
 import re
-import time
 from typing import get_args
 
 import numpy as np
@@ -16,7 +15,6 @@ from pyvista.core.errors import DeprecationError
 from pyvista.core.errors import PyVistaDeprecationWarning
 from pyvista.core.filters.image_data import _InterpolationOptions
 from tests.conftest import NUMPY_VERSION_INFO
-from tests.conftest import flaky_test
 
 BOUNDARY_LABELS = 'boundary_labels'
 MORPHOLOGICAL_MAX_VAL = 42.0
@@ -419,19 +417,10 @@ def test_contour_labels_cell_data(channels):
     assert voxel_surface_contoured.n_cells == voxel_surface_extracted.n_cells
 
 
-@flaky_test
 @pytest.mark.needs_vtk_version(9, 3, 0)
 def test_contour_labels_strict_external(channels):
-    start = time.perf_counter()
-    with pytest.warns(pv.PyVistaDeprecationWarning):
-        channels.contour_labels('external', orient_faces=False)
-    time_slow = time.perf_counter() - start
-
-    start = time.perf_counter()
     with pytest.warns(pv.PyVistaDeprecationWarning):
         contours = channels.contour_labels('strict_external', orient_faces=False)
-    time_fast = time.perf_counter() - start
-    assert time_fast < time_slow / 1.5
 
     # Test output is simplified correctly
     assert contours.active_scalars.ndim == 1

